content/en/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md

```diff
@@ -1,76 +1,201 @@
 ---
 reviewers:
 - fgrzadkowski
-- piosz
-title: Resource metrics pipeline
+- piosz 
+title: Resource metrics pipeline 
 content_type: concept
 ---
 
 <!-- overview -->
 
-Resource usage metrics, such as container CPU and memory usage,
-are available in Kubernetes through the Metrics API. These metrics can be accessed either directly
-by the user with the `kubectl top` command, or by a controller in the cluster, for example
-Horizontal Pod Autoscaler, to make decisions.
+Kubernetes supports a built-in resource metrics pipeline. A fixed set of metrics consisting of node and pod CPU and memory usage for Kubernetes built-in workload autoscaling pipelines. 
 
-<!-- body -->
+The [HorizontalPodAutoscaler (HPA)](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/) and [VerticalPodAutoscaler (VPA)](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler#readme) use these metrics to adjust workload replicas and resources to meet customer demand.
 
-## The Metrics API
+You can also view the resource metrics using the [`kubectl top`](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#top) command.
 
-Through the Metrics API, you can get the amount of resource currently used
-by a given node or a given pod. This API doesn't store the metric values,
-so it's not possible, for example, to get the amount of resources used by a
-given node 10 minutes ago.
+{{< note >}}
+The resource metrics pipeline only supports CPU and memory metrics for use by HPA and VPA. To access a more complete set of metrics, you will need to deploy a [Full metrics pipeline](https://kubernetes.io/docs/tasks/debug-application-cluster/resource-usage-monitoring/#full-metrics-pipeline). 
```

Comment:
```suggestion
The Metrics API, and the metrics pipeline that it enables, only offers the minimum
CPU and memory metrics to enable automatic scaling using HPA and / or VPA.
If you would like to provide a more complete set of metrics, you can complement
the simpler Metrics API by deploying a second
[metrics pipeline](/docs/tasks/debug-application-cluster/resource-usage-monitoring/#full-metrics-pipeline)
that uses the _Custom Metrics API_.
```

Reply:
`If you would like to provide a more complete set of metrics` -> `If you would like to autoscale based on other metrics`